content/news/2020-10-04-wallpaper-contest.md

```diff
@@ -0,0 +1,40 @@
++++
+type = "news"
+title = "Call for Wallpapers"
+author = "bitigchi"
+date = "2020-10-08 13:00:00+03:00"
+tags = ["wallpapers", "contest"]
++++
+
+One of the most common feedback that we receive is that Haiku needs some nice colourful wallpapers shipping with its releases. We definitely want to improve the current situation. And we need your help.
+
+Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned at fixed coordinates on the screen. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
+It's also works relatively well with different screen resolutions.
+
+We are looking for artists that can contribute to Haiku with some appealing wallpapers. To be more precise, we are looking for either:
+
+1. Artistic variations of the Haiku logo, to be used as the default wallpaper
+2. Vector or bitmap illustrations, to be included in the release
+3. We are NOT looking for photographs of any kind, any submitted work should be an illustration of some kind.
+
+## Guidelines
+
+Artists are free to submit in any category they please. We hope the guidelines below will help:
+
+- Illustrations should be positioned in landscape, and should be at least 5K in size (for bitmap submissions).
+- Logo variations are encouraged to preserve the current theme. Doodle versions are welcome.
+- Using subtle haiku themes, such as leaves, Far-Eastern typography, and literature are encouraged. Anime-themed drawings are also welcome, but should there be human characters, they should not be the main focus.
+- If you want to go full 90's, and submit a version suitable for tiled backgrounds, just let us know :).
+- Entries should include a note that the work is licensed under [CC Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/) or a similar license.
+- Artists are encouraged to name their work as they please, otherwise you would be leaving the naming part to the mercy of the Haiku developers.
+- Submitted work should be in `*.wonderbrush`, `*.svg`, `*.kra` or `*.xcf` format.
```

Comment:
Should we add something like:

```suggestion
- Submitted work should be in `*.wonderbrush`, `*.svg`, `*.kra` or `*.xcf` format (or whatever is the native workfile format of your graphics app).
```

Reply:
I personally think allowing this (hence proprietary applications) is not a good idea. Keeping it FOSS allows everyone to access these files without any issues.

Even WonderBrush is a stretch, since it's not available on 64-bit. :))

Reply:
As a Haiku user :)  I'm not that familiar with supported file exports of professional graphics apps. Do Photoshop, Illustrator et al. export well in these formats?

Reply:
Photoshop cannot open/save in `.xcf`, but GIMP can open Photoshop files, so they can save it as `.xcf`. Illustrator has perfect `*.svg` support.

Reply:
Very well, I'm persuaded. :)
Anything else? Otherwise, I'll point people to this PR on IRC for last input, and merge tomorrow.

Reply:
I think that's all from my side. Thanks!